website/docs/r/app_service_custom_hostname_binding.html.markdown

```diff
@@ -62,6 +62,8 @@ The following arguments are supported:
 
 * `app_service_name` - (Required) The name of the App Service in which to add the Custom Hostname Binding. Changing this forces a new resource to be created.
 
+-> **NOTE:** As the App Service is a family of services and contains the function app, so Custom Hostname Binding also can be added into the function app.
```

Comment:
this wants moving up & integrating into the description above, since it's not discoverable at this level - will push a commit for this